packages/near-fast-auth-signer/src/lib/controller.ts

```diff
@@ -146,13 +155,31 @@ class FastAuthController {
 
   async signDelegateAction({ receiverId, actions, signerId }) {
     this.assertValidSigner(signerId);
-
-    const account = new Account(this.connection, this.accountId);
-    return account.signedDelegate({
-      actions,
-      blockHeightTtl: 60,
-      receiverId,
-    });
+    try {
+      // webAuthN supported browser
+      const account = new Account(this.connection, this.accountId);
+      return account.signedDelegate({
+        actions,
+        blockHeightTtl: 60,
+        receiverId,
+      });
+    } catch {
+      // fallback, non webAuthN supported browser
```

Comment:
Just wondering, instead of using try/catch for a known scenario. Can we check if the browser supports webAuthN and then chose the code to run?

Reply:
The specific condition we are checking for is whether there is a key in the controller. The controller is indifferent to whether the browser supports webauthn or not, it just tries to sign the delegate locally and if that fails, it falls back to using the recovery service. We could possibly be more explicit by checking if we have a key in the local keystore before trying to sign here but the current logic is fine too wdyt